docs/user/openstack/add_worker_nodes_by_hand.md

```diff
@@ -0,0 +1,44 @@
+#Adding workers nodes by hand
+The prefered method of adding worker nodes to an OpenShift cluster is
+to use the a worker machineset and machine api operator as described in
+https://github.com/openshift/machine-api-operator.
+
+It is also possible to add a worker through other tools like ansible. For example, 
+the workers may be added as described in [Installing OpenShift on OpenStack User-Provisioned Infrastructure][1];
+or when scaling up the workers using [Openshift-ansible][2]
+
+When doing so, there are a few important points that should be taken into account. 
+
+##Apply Necessary Security Groups To Nodes
+When adding a new worker node, the necessary security groups need to be applied so the node is 
+allowed access the cluster API endpoints and services.   
+
+If the node being added is attached the cluster os network of an existing cluster,  the pre-existing worker security groups maybe 
```

Comment:
"os network" os a pretty obscure term. Probably should use something else more descriptive. os network refers to the openstack network being used to connect the nodes together.